packages/grafana-data/src/events/pageBanner.ts

```diff
@@ -0,0 +1,27 @@
+import { BusEventWithPayload } from './types';
+
+/** @alpha */
+export class PageBannerDisplayEvent extends BusEventWithPayload<PageBannerEventPayload> {
+  static type = 'page-banner-display';
+}
+
+/** @alpha */
+export type PageBannerEventPayload = {
+  text: string;
+  closable?: boolean;
```

Comment:
I don't think that `closable` is being used anywhere? 

Passed the data as false and still, the X button is there.